doc/en/components/grids/_shared/export-excel.md

```diff
@@ -0,0 +1,234 @@
+---
+title: {Platform} {Component} Export to Excel - {ProductName} for {Platform}
+_description: With {ProductName} Excel Exporter you can make client Excel functionality more convenient & simpler. This format allows features like filtering, sorting, etc.
+_keywords: {Platform}, {ComponentTitle}, {ComponentTitle}, {ProductName}, Infragistics
+mentionedTypes: [{ComponentApiMembers}]
+sharedComponents: ["Grid", "TreeGrid", "HierarchicalGrid"]
+---
+
+# {Platform} {Component} Export to Excel Service
+
+
+<p class="highlight">
+  The Excel Exporter service can export data to excel from the {Component}. The data export functionality is encapsulated in the [`ExcelExporterService`]({environment:demosBaseUrl}/classes/excelexporterservice.html) class and the data is exported in MS Excel table format. This format allows features like filtering, sorting, etc. To do this you need to invoke the [`ExcelExporterService`]({environment:demosBaseUrl}/classes/excelexporterservice.html)'s [`export`]({environment:demosBaseUrl}/classes/excelexporterservice.html#export) method and pass the {ComponentTitle} component as first argument to export grid easily.
+</p>
+
+## {Platform} Excel Exporter Example
+
+<!-- ComponentStart: Grid -->
+
+<code-view style="height: 800px;" 
+           data-demos-base-url="{environment:demosBaseUrl}"
+           explicit-editor="stackblitz"
+           iframe-src="{environment:demosBaseUrl}/services/export-excel-sample-1" alt="Blazor Excel Exporter Example">
+</code-view>
+
+<!-- ComponentEnd: Grid -->
+
+<!-- ComponentStart: TreeGrid -->
+
+<code-view style="height: 350px;" 
+           data-demos-base-url="{environment:demosBaseUrl}"
+           explicit-editor="stackblitz"
+           iframe-src="{environment:demosBaseUrl}/services/export-excel-tree-grid-sample" alt="Blazor Excel Exporter Example">
+</code-view>
+
+<!-- ComponentEnd: TreeGrid -->
+
+<!-- ComponentStart: HierarchicalGrid -->
+
+<code-view style="height: 600px;" 
+        data-demos-base-url="{environment:demosBaseUrl}"
+        explicit-editor="stackblitz"
+        iframe-src="{environment:demosBaseUrl}/hierarchical-grid/hierarchical-grid-excel-export" alt="Blazor Excel Exporter Example">
+</code-view>
+
+<!-- ComponentEnd: HierarchicalGrid -->
```

Comment:
replace above 3 code-view blocks with just one code-view and remove component build flag because routing path to the sample will be the same for all grid samples in Sample Browser.